src/components/Weather/Weather.jsx

```diff
@@ -1,8 +1,63 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const APIKEY = "40d6f448e392001b5598241621cd7b46";
+
 function Weather() {
+	const [city, setCity] = useState("");
```

Comment:
suggestion: you can extract the logic in the WeatherContainer.js using logic-presentation layer design.